### ISSUE TITLE
project cluttering

### ISSUE THREAD

**Issue by Phoscur** [webpack/webpack#42]
I'm not sure what the best solution for this is, but IDEs like Aptana Studio have a file indexer, installing a lot of modules slows down the build. Pretty annoying if you have to wait before you can do something else.
That's just partially webpack's problem, but it could still use some refinement.

webpacks tutorial tells you to install webpack locally and globally. Shouldn't it be just one of those? Would be nice if the user could choose between installing his dev environment separately (global) or bound to the project. The latter makes more sense at the moment, as webpack introduces some specific features, which binds one's project to webpack anyways.

Still, I would prefer a complete global install of webpack and it's modules.
For comparison, grunt installs globally, while it's plugins are installed locally. That at least, keeps that huge chunk of grunt main code out of the project, while plugins are small. That is, I think, because you can't put any modules in your package.json to be installed globally.

To sum up: I would like to see webpack being more loosely coupled to my project and satisfied with a global install. One build.json or webpack.js in the projects main should be sufficient.
